src/librustc_resolve/late/lifetimes.rs

```diff
@@ -347,9 +347,7 @@ fn krate(tcx: TyCtxt<'_>) -> NamedRegionMap {
             lifetime_uses: &mut Default::default(),
             missing_named_lifetime_spots: vec![],
         };
-        for (_, item) in &krate.items {
-            visitor.visit_item(item);
-        }
+        krate.items.values().for_each(|item| visitor.visit_item(item))
```

Comment:
I prefer avoiding `for_each` as an anti-pattern, tbh, but the `.values()` change seems fine.

Reply:
sure, I'll change it